### DISCUSSION TITLE
Creator Royalties  🎨💰

### DISCUSSION THREAD

**Discussion by WILDM4N** [ordinals/ord#773] [Ideas]
Should there be a [suggested royalty] field in the ordinal inscription?

I believe this is a complex and nuanced topic that warrants discussion.

**Comment by casey** [comment]
Royalties as an idea are fine. However, all implementations of royalties that I have seen have one or more of the following problems:

1. Are not enforceable, leading to user confusion and a race to the bottom, as market realities become clear
2. Make markets less efficient
3. Impose arbitrary or draconian restrictions on users and limit their property rights
4. Are technically complex

As such, I believe royalties are perhaps reasonable in theory, but have no practically realizable implementation with more upsides than downsides, and that optional or suggested royalties just cause user confusion, and can be negotiated at a higher layer, and thus need no representation in the base layer.
